docs/antora.yml

```diff
@@ -0,0 +1,7 @@
+name: ios
+title: ownCloud iOS App Documentation
```

Comment:
This title gets displayed within the sidebar navigation, don't you think this is somehow redundant and long?

Reply:
Might be, now that you mention it.

Reply:
I've just corrected that, making the name shorter.